README.rst

```diff
@@ -54,24 +54,29 @@ PyVista
 .. |python| image:: https://img.shields.io/badge/python-3.7+-blue.svg
    :target: https://www.python.org/downloads/
 
-
-+----------------------+-----------+------------+
-| Deployment           | |pypi|    |   |conda|  |
-+----------------------+-----------+------------+
-| Build Status         | |GH-CI|   |  |python|  |
-+----------------------+-----------+------------+
-| Metrics              | |codacy|  |  |codecov| |
-+----------------------+-----------+------------+
-| Activity             | |PyPIact| | |condaact| |
-+----------------------+-----------+------------+
-| Citation             | |joss|    |  |zenodo|  |
-+----------------------+-----------+------------+
-| License              | |MIT|     |            |
-+----------------------+-----------+------------+
-| Community            | |slack|   |  |discuss| |
-+----------------------+-----------+------------+
-| Formatter            | |black|   |  |isort|   |
-+----------------------+-----------+------------+
+.. |Sponsored by NumFOCUS| image:: https://img.shields.io/badge/sponsored%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
+   :target: https://numfocus.org
```

Comment:
I'm curious why this was merged with the "Sponsored by" language still present? This should be changed to:

```suggestion
.. | Affiliated with NumFOCUS| image:: https://img.shields.io/badge/affiliated-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
   :target: https://numfocus.org
```